### ISSUE TITLE
Wrong indentation with allman braces and mixins

### ISSUE THREAD

**Issue by pascaliske** [sasstools/sass-lint#989]
I found an error with the indentation rule of `sass-lint`. In our project we want to use the "allman" brace style and 4 spaces for indentation. If we setup sass-lint to check the indentation and the braces we get lots of errors for wrong indentations. 

**What version of Sass Lint are you using?**

```bash
$ npm list sass-lint
sass-lint@1.10.2 
```
**Please include any relevant parts of your configuration**

```yml
rules:
  # we want "allman" braces -> always on a new line
  brace-style:
    - 1
    - style: allman
    - allow-single-line: false

  # always indent using 4 spaces
  indentation:
    - 1
    - size: 4
```

**What did you do? Please include the actual source code causing the issue.**

```scss
.m-module
{
    ...
    @include use-breakpoint(mobile, below)
    {
        position: relative;
        margin: 0 20px;
    }
    ...
}

```
**What did you expect to happen?**

No linting error on line 4 in the above snippet.

**What actually happened? Please include any error messages given to you by Sass Lint.**

I got this linting error from the sass-lint cli:
```bash
@4 Expected indentation of 8 spaces but found 4. (indentation)
```

**Comment by DanPurdy** [comment]
Sorry, indentation is still a little buggy and I've not had time to work on it recently. There's still a few outstanding issues with indentation here. I'll hopefully get some time to take a look soon but I'm not sure when.
